README.md

```diff
@@ -1,8 +1,44 @@
-# starknet-devnet-rs
+<!-- logo / title -->
+<p align="center" style="margin-bottom: 0px !important">
+  <img width="200" src="https://user-images.githubusercontent.com/2848732/193076972-da6fa36e-11f7-4cb3-aa29-673224f8576d.png" alt="Devnet-RS" align="center">
+</p>
 
-A local testnet for Starknet... in Rust
+<h1 align="center" style="margin-top: 12px !important">Starknet Devnet RS</h1>
 
-This repository is work in progress, please be patient. As of Starknet 0.12.2, [Pythonic Devnet](https://github.com/0xSpaceShard/starknet-devnet) is still the way to go.
+<p align="center" dir="auto">
+  <a href="https://hub.docker.com/r/shardlabs/starknet-devnet-rs/tags" target="_blank">
+    <img src="https://img.shields.io/badge/dockerhub-images-important.svg?logo=Docker" style="max-width: 100%;">
+  </a>
+  <a href="https://starkware.co/" target="_blank">
+    <img src="https://img.shields.io/badge/powered_by-StarkWare-navy" style="max-width: 100%;">
+  </a>
+</p>
+
+A local testnet for Starknet... in Rust!
+
+This repository is work in progress, please be patient. Please check below the status of features compared with the [Pythonic Devnet](https://github.com/0xSpaceShard/starknet-devnet):
+
+### Supported Features
+
+- [x] `blockifier` version v0.3.0-rc0
+- [x] RPC v0.4.0
+- [x] [Dump & Load](https://github.com/0xSpaceShard/starknet-devnet-rs#dumping--loading)
+- [x] [Mint token - Local faucet](https://0xspaceshard.github.io/starknet-devnet/docs/guide/mint-token)
+- [x] Starknet.js test suite passes 100%
+
+### TODO
```

Comment:
We are currently tracking progress using issues and the PR template reminds us to link closeable. If we are going to introduce a new feature list, we need to remember to edit it when needed, so I propose editing the PR template by introducing a new bullet that will remind us to edit this new TODO section.